ongebo/fff-frontend
# #163673577 Integrate Coveralls.io for test coverage reporting

### What does this PR do?
Sets up the project for coverage reporting to Coveralls.io
#### Description of the task to be completed?
- Install coveralls in package.json
- Add coveralls script to package.json
- Update .travis.yml to push stats to Coveralls
- Add coverage status badge to README.md
#### How should this be manually tested?
- Push commits to the repository
- Click on the coverage badge in README.md to verify that stats were pushed to Coveralls
#### What are the relevant Pivotal Tracker stories?
[#163673577](https://www.pivotaltracker.com/story/show/163673577)
